api/inc/api.h

```diff
@@ -90,6 +90,7 @@ typedef struct {
     void (*spin_unlock)(UvisorSpinlock * spinlock);
 
     void (*debug_semihosting_enable)(void);
+    void (*debug_print_enable)(bool);
```

Comment:
this function comes to solve initialization order problems when using debug driver.
I believe this logic should reside on application side. An alternative is to look for any falg/callback from mbed-os - for checking such a condition.

Application can define a global `bool debug_print_enable = false;` variable, and avoid printing if it is disabled.
as i see it:
```
bool debug_print_enable = false;
void debug_print_driver(THaltError error_code, const THaltInfo *bsod)
{
    if (debug_print_enable){
        printf("oops\n");
    }
}
int main(void)
{
    debug_print_enable = true;

    ...

}